content/home.md

```diff
@@ -0,0 +1,42 @@
+type: page
+slug: home
+body_class: home
+title: Home
+---
+
+PyCon UK is the annual gathering of the UK Python community and its friends
+from around the world.  This year, the conference is held at [Cardiff City
+Hall](http://www.cardiffcityhall.com/), from Thursday 15th to Monday 19th
+September.
+
+We&rsquo;ll be putting on a programme of talks, workshops, and other events
```

Comment:
We have charset set to utf-8 in the base template so yea I think we just drop all the escape codes, unless there's something in wok that might cause issues @inglesp?